cmd/dep/root_analyzer_test.go

```diff
@@ -19,3 +24,90 @@ func TestRootAnalyzer_Info(t *testing.T) {
 		}
 	}
 }
+
+func TestLookupVersionForLockedProject_MatchRevisionToTag(t *testing.T) {
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+
+	ctx := newTestContext(h)
+	sm, err := ctx.SourceManager()
+	h.Must(err)
+	defer sm.Release()
+
+	pi := gps.ProjectIdentifier{ProjectRoot: gps.ProjectRoot("github.com/sdboyer/deptest")}
+	rev := gps.Revision("ff2948a2ac8f538c4ecd55962e919d1e13e74baf")
+	v, err := lookupVersionForLockedProject(pi, nil, rev, sm)
+	h.Must(err)
+
+	wantV := "v1.0.0"
+	gotV := v.String()
+	if gotV != wantV {
+		t.Fatalf("Expected '%s', got '%s'", wantV, gotV)
```

Comment:
How about adding a little context to the failure message, like we try to do everywhere and in alignment with the existing test in the file?
Same for the tests below 🙂